versioned/gc/iceberg-actions/pom.xml

```diff
@@ -30,11 +30,20 @@
 
   <name>Nessie - Versioned - GC for Spark - Iceberg - Actions</name>
 
-  <properties>
-    <iceberg.nessie.version>${project.version}</iceberg.nessie.version>
-  </properties>
-
   <dependencies>
+    <dependency>
+      <groupId>org.projectnessie</groupId>
+      <artifactId>nessie-client</artifactId>
+      <version>${iceberg.nessie.version}</version>
+      <scope>test</scope>
```

Comment:
why did we have to add this

Reply:
are you referring to the exclusions? @snazy ran into a very similar problem as I did. He had breaking changes in the model classes and so that means we need to use the correct model classes here

Reply:
I was referring to the entire dependency.

Reply:
see my comment on https://github.com/projectnessie/nessie/pull/1783#discussion_r691368847. It doesn't work without forcing the `nessie-client` version

Reply:
all we ddi was move a property to the high level pom. Why did it require the introduction of new dependencies ot the module?